presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/event/BenchmarkQueryEvent.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.event;
+
+import com.facebook.airlift.event.client.EventField;
+import com.facebook.airlift.event.client.EventType;
+import com.facebook.presto.jdbc.QueryStats;
+import io.airlift.units.Duration;
+
+import javax.annotation.concurrent.Immutable;
+
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+@Immutable
+@EventType("BenchmarkQuery")
+public class BenchmarkQueryEvent
+{
+    public enum EventStatus
+    {
+        SUCCEEDED,
+        FAILED
+    }
```

Comment:
Let's just name this enum `Status`. For nested classes, their name don't need to include parts that are already described by the outer class.